distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -264,4 +432,63 @@ private void showServerReply(FTPClient ftpClient) {
             }
         }
     }
-}
+
+    /**
+     * Sets the clientAuth configuration in catalog-ftp feature.
+     *
+     * @throws Exception
+     */
+    private void setClientAuthConfiguration(String clientAuth) throws Exception {
+        Configuration config =
+                getAdminConfig().getConfiguration("ddf.catalog.ftp.FtpServerStarter");
+        config.setBundleLocation("mvn:ddf.catalog/ftp/" + System.getProperty("ddf.version"));
+        Dictionary properties = new Hashtable<>();
+
+        properties.put("port", FTP_DEFAULT_PORT);
+        properties.put("clientAuth", clientAuth);
+        config.update(properties);
+
+        switch (clientAuth) {
+        case CLIENT_AUTH_WANT:
+            expect("SSL handshake to succeed with FTPS client without certificate because clientAuth = \"want\"").within(
+                    SET_CLIENT_AUTH_TIMEOUT_SEC,
+                    TimeUnit.SECONDS)
+                    .until(() -> {
+                        FTPSClient client = null;
+                        try {
+                            client = createSecureClient(false);
+                            disconnectClient(client);
+                            return true;
+                        } catch (SSLException e) {
+                            //SSL handshake failed
+                            disconnectClient(client);
+                            return false;
+                        } catch (SocketException | FTPConnectionClosedException e) {
+                            //connection failed
+                            return false;
+                        }
+                    });
+            break;
+        case CLIENT_AUTH_NEED:
+            expect("SSL handshake to fail with FTPS client without certificate because clientAuth = \"need\"").within(
+                    SET_CLIENT_AUTH_TIMEOUT_SEC,
+                    TimeUnit.SECONDS)
+                    .until(() -> {
+                        FTPSClient client = null;
+                        try {
+                            client = createSecureClient(false);
+                            disconnectClient(client);
+                            return false;
```

Comment:
I added some more comments to explain why this wait condition for clientAuth=NEED should wait for a SSLException